Model/Currency.hs

```diff
@@ -1,3 +1,9 @@
+{-# LANGUAGE CPP #-}
+
+#if MIN_VERSION_base(4,8,0)
+{-# LANGUAGE FlexibleContexts #-}
```

Comment:
I couldn't find any documentation on the extension except for GHC 7.10. GHC isn't exactly known for its documentation, though.